python/tvm/relay/frontend/pytorch.py

```diff
@@ -3244,7 +3267,8 @@ def get_all_op_names(graph):
     return set(node.kind() for node in nodes)
 
 
-def from_pytorch(script_module, input_shapes, custom_convert_map=None, default_dtype="float32"):
+def from_pytorch(script_module, input_shapes, custom_convert_map=None, default_dtype="float32",
+                 input_types=None):
```

Comment:
I must admit that I'd find it more natural to have the types with the shapes as an optional third item in the tuple.
But this is a matter of style and if you think differently, feel free to ignore.

Reply:
yeah, I also prefer stuffing types into the tuple (name, (shape, type)) and rename input_shapes to something like input_info or something

Reply:
Just for forward compatibility. If not, one argument `input_info` maybe the best choice.

Reply:
I think if it doesn't break existing code, it is fine to change the api a bit.